### ISSUE TITLE
V14 Signs are bugged

### ISSUE THREAD

**Issue by KumikoTsukimi** [lgow/endofherobrine#34]
Hello I have recently really enjoyed the modpack and been playing it with a few friends on a server, We all did realise quickly tho that If Herobrine places a sign Its bugged and It wont show the text but insted say stuff like "sign.typed3.line1" and so on..

![im34343141age](https://github.com/lgow/endofherobrine/assets/172211623/d50a96d1-8754-4314-a02e-0e912a2470d3)

Im guessing there should be text showing up. But it sadly doesnt for some reason

**Comment by FizzyAndyboy** [comment]
yeah im playing Fear of nightfall mod pack and its the same issue
